test/java/seedu/addressbook/logic/LogicTest.java

```diff
@@ -99,6 +99,27 @@ public void execute_unknownCommandWord() throws Exception {
         assertCommandBehavior(unknownCommand, HelpCommand.MESSAGE_ALL_USAGES);
     }
 
+    @Test
+    public void execute_displayweek() throws Exception {
+        // setup expectations
```

Comment:
This seems a bit janky of a test. 
Because if the test/code is not done correctly, it may fail some times and pass some times, due to the fact that it is querying the current date. In an ideal test, you should be testing a fix state of a system rather than a variable state. The reason is, you want to ensure that the desired outcome will happen in a given state 100% of the time in test cases.
So instead, you should inject the date to be tested into the test case instead of using the global date

Note: This may not be the focus of CS2113, but it is a tiny issue on test case design principles.